addons/mail/models/mail_scheduled_message.py

```diff
@@ -332,7 +332,7 @@ def _post_messages_cron(self, limit=50):
     def _to_store_defaults(self):
         return [
             Store.Many("attachment_ids"),
-            Store.One("author_id", rename="author"),
+            Store.One("author_id"),
```

Comment:
According to the diff, only `name`, `userId` and `avatar_128` are used?
We might want to list the fields explicitly instead of sending many extra stuff.